components/card/card.html

```diff
@@ -0,0 +1,208 @@
+            <h2 class="grey mb2">Cards with badges</h2>
+            <div class=" card card1">
+                <div class="card-badge">
+                    <span class="sale-badge">on Sale</span>
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
+                    <div class="card-text"> 
+                        <h3>T-shirt</h3>
+                        <div> cotton T-shirt with a white logo on it.very comfortable to wear.Available in all sizes.
+                        </div>
+                    </div>
+                    <footer class="footer">
+                        <h4>Rs. 300</h4> <button>Buy</button>
+                    </footer>
+                </div>
+            </div>
+            <br><br>
+
+            <div class="iframe-code pb1 flex-column">
+                <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=vscode&wt=none&l=auto&width=680&ds=true&dsyoff=61px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522%2520card%2520card1%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-badge%2522%253E%250A%2520%2520%2520%2520%253Cspan%2520class%253D%2522sale-badge%2522%253Eon%2520Sale%253C%252Fspan%253E%250A%2520%2520%2520%2520%253Cimg%2520class%253D%2522card-image%2522%2520src%253D%2522https%253A%252F%252Fimages.unsplash.com%252Fphoto-1583743814966-8936f5b7be1a%253Fixlib%253Drb-1.2.1%2526ixid%253DMnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%25253D%25253D%2526auto%253Dformat%2526fit%253Dcrop%2526w%253D400%2526q%253D60%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-text%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ch3%253ET-shirt%253C%252Fh3%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%253E%2520cotton%2520T-shirt%2520with%2520a%2520white%2520logo%2520on%2520it.very%2520comfortable%2520to%2520wear.Available%2520in%2520all%2520sizes.%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cfooter%2520class%253D%2522footer%2522%253E%2520%253Ch4%253ERs.%2520300%253C%252Fh4%253E%2520%253Cbutton%253EBuy%253C%252Fbutton%253E%253C%252Ffooter%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A"
+                    style=" height: 200px;  border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+            <!--Card 2-->
+           
+            <h2 class="grey mb2">Cards with badge and dismiss</h2>
+            <div class=" card card2">
+                <div class="card-badge">
+                    <span class="sale-badge">on Sale</span>
+                    <button class="dismiss-badge">X</button>
```

Comment:
use cross icon here

Reply:
i have added the icons,thankyou for the review